packages/ozone/src/db/migrations/20250415T201720309Z-verification.ts

```diff
@@ -0,0 +1,25 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('verification')
+    .addColumn('uri', 'text', (col) => col.primaryKey())
+    .addColumn('issuer', 'text', (col) => col.notNull())
+    .addColumn('subject', 'text', (col) => col.notNull())
+    .addColumn('handle', 'text', (col) => col.notNull())
+    .addColumn('displayName', 'text', (col) => col.notNull())
+    .addColumn('revokeReason', 'text')
+    .addColumn('revokedBy', 'text')
+    .addColumn('revokedAt', 'text')
+    .addColumn('createdAt', 'text', (col) =>
+      col.defaultTo(sql`now()`).notNull(),
+    )
```

Comment:
Is this the `createdAt` time contained in the record or ozone's initial indexed time?

Reply:
this is meant to be ozone's index time but i see the record itself contains a `createdAt` timestamp so I'll use that.